modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,149 @@
+= Migrate Client Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: migrate client applications, migrate applications, migrate 
+
+You might have client credentials of applications residing outside of Anypoint Platform if you were previously using a different API management. You must first migrate those client applications, with their client ID and client secret credentials, before you can start using Anypoint Platform.
+
+Because client IDs are unique in Anypoint Platform, you cannot reuse the same client ID in two different accounts. Your client applications can consume APIs whose organization is registered with a Anypoint Platform client provider or an external client provider. The instructions for these two scenarios differ:
```

Comment:
The first sentence leads me to expect some sort of comment about how to avoid duplicating credentials during migration, but that comment does not exist. 
The second sentence does not build on the first; it repeats the opening paragraph above.
The third sentence refers to "instructions" for "two scenarios": instructions for doing what? which two scenarios?

Reply:
Done! Thanks!